server/router/src/test/java/org/infinispan/server/router/utils/HotRodClientTestingUtil.java

```diff
@@ -0,0 +1,52 @@
+package org.infinispan.server.router.utils;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+
+import java.net.InetAddress;
+
+public enum HotRodClientTestingUtil {
```

Comment:
Why do you need this? Can't use/enhance `org.infinispan.client.hotrod.test.HotRodClientTestingUtil` ?